ui/pages/swaps/smart-transaction-status/smart-transaction-status.js

```diff
@@ -441,6 +451,7 @@ export default function SmartTransactionStatusPage() {
         {description && (
           <Text
             variant={TextVariant.bodySm}
+            as="h6"
             boxProps={{ ...(blockExplorerUrl && { margin: [1, 0, 0] }) }}
```

Comment:
Need to update boxProps here to use `marginTop={1}`

Reply:
Hey @georgewrmarshall ,

Am I supposed to do this?
![Screenshot (78)](https://user-images.githubusercontent.com/125105825/235889414-597bc749-8435-42e6-99f8-3a3f756727d9.png)

Kindly look into this.